docs/news/news-v5.0.x.rst

```diff
@@ -66,20 +66,25 @@ Open MPI version 5.0.0rc13
                 Libevent symbols and then statically pulled the
                 library into ``libmpi.so``.
 
-- Changes since rc12:
-
-  - Update PMIx to the ``v4.2.6`` release tag. Hash: ``f20e0d5``.
-  - Update PRRTE to the ``v3.0.1`` release tag. Hash: ``63370ca``.
-  - Lots of documentation updates.
-  - Fixed parameter name in ``MPI_Intercomm_merge``. Thanks to Yan Wu for the report.
-  - ``OFI``: Update NIC selection to determine optimal interfaces from the current process.
-  - Fix reordering of received data in ``MPI_Gather``.
-  - Disable builds with ``HWLOC`` versions >= 3.0.0. This is currently not supported.
-  - Fix re-ordering of ranks in ``MPI_Dist_graph_create``.
-  - ``coll/HAN``: Fix bug when using ``MPI_IN_PLACE`` with ``MPI_Reduce``.
-  - Fix ``MPI_Type_Dup`` to propagate errors from inner calls.
-  - Fix the compilation of the monitoring infrastructure.
-  - Various other bug fixes.
+- Changes since rc13:
+
+  - Update PMIx to hash: ``f8f578392ec77dd7a1d76ca697da4f15afcb0161``.
+  - Update PRRTE to hash: ``bb4085053a0b268ae2a2e04ed56387f53e4a3e7a``.
+  - Documentation updates
+  - Fix build case with --disable-prrte
+  - Update PRRTe and PMIx pointers to pull in fixes, including spurious log messages, and also
+    RPM fixes.
```

Comment:
I think this is removed, are you looking at the overall changes or commit by commit?

Reply:
Yeah, I was looking commit-by-commit and I ran out of time, so I submitted what I had done so far.  It would have been nice to squash this down to a single commit, but no prob -- merged and done for now.

Reply:
Yeah I agre..I probably should have done that. Sigh, my fault.

Reply:
Some of these were back ports of things that were missed on main..already in v5.0x, so it's not _quite_ as bad as it looks.